pkg/scheduler/factory/factory.go

```diff
@@ -293,6 +294,12 @@ func NewConfigFactory(
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
 		// Setup volume binder
 		c.volumeBinder = volumebinder.NewVolumeBinder(client, pvcInformer, pvInformer, nodeInformer, storageClassInformer)
+
+		storageClassInformer.Informer().AddEventHandler(
+			cache.ResourceEventHandlerFuncs{
+				AddFunc: c.onStorageClassAdd,
```

Comment:
How about deleting a storage class or updating it? Do they affect scheduling decisions as well? If so, do we need event handlers for them as well?

Reply:
If my understanding of the pod queue is correct, we're only concerned about moving pods from unschedulable -> active.  So deleting or updating a storageclass could cause predicates to fail, but not to pass.  So it wouldn't cause an unschedulable pod to become schedulable.

Reply:
Oh, but update/delete might cause predicates that previously passed to fail.  In that case, we should invalidate the ecache.

Reply:
Pushed update that adds delete handler.  Update is not needed.